scripts/whisper/rknn/test_qnn.py

```
@@ -0,0 +1,117 @@
+    tokenizer = whisper.tokenizer.get_tokenizer(
+        model.is_multilingual, num_languages=model.num_languages
+    )
+
+    model.eval()
+
+    cross_kv_pair = []
+    for i in range(4):
+        k = features = np.fromfile(f"./cross_k_{i}.raw", dtype=np.float32).reshape(
+            1, 1500, 384
+        )
+        v = features = np.fromfile(f"./cross_v_{i}.raw", dtype=np.float32).reshape(
```

Flake8 (7.3.0)

[error] local variable 'features' is assigned to but never used

(F841)